### PR TITLE
HPCC-17317 DOCS:ECLWatch Add Delete Part

### PR DESCRIPTION
**Pull request by g-pan**
Fix HPCC-17317 DOCS:ECLWatch Add Delete Part
Add documentation to the ECL Watch doc on
using the Roxie Package Maps, Add, Delete, Activate Part
New Pull request, Replacing Pull Request # 10030, 10050 & 10056 - due to
substantial changes and some technical difficulties

@JamesDeFabia  please review
@miguelvazq please review
@afishbeck please review

For convenience PDF can be reviewed at: 
http://10.240.32.243/view/Docs/job/DocDev05/CE=ubuntu-14.04-amd64-1/lastSuccessfulBuild/artifact/build-docs/Release/docs/The_ECL_Watch_Manual-6.4.0-rc1.pdf
 resulting from successful test build.

Signed-off-by: g-pan <greg.panagiotatos@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [x] I have added tests to cover my changes.
http://10.240.32.243/view/Docs/job/DocDev05/CE=ubuntu-14.04-amd64-1/
  - [x] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [x] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

### CODE REVIEW

File: docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml [hunk 1]
```diff
@@ -589,33 +597,299 @@ depending on where they wind up.-->
...
+              <para>Press the <emphasis role="bold">Validate</emphasis> button
+              to validate the package map.</para>
+            </listitem>
+          </orderedlist>
+
+          <para>The result is shown on the <emphasis role="bold">Validate
+          Active Package Map</emphasis> tab. You can validate any package map,
+          active or inactive.</para>
+
+          <para>To validate any package map:</para>
```
Comment (afishbeck) [inline docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml]: Not sure on exact wording, but the key is these are packagemaps that aren't yet on the system.

Something like "To validate an external packagemap"
and/or "a packagemap not yet uploaded onto the environment."

**Comment by afishbeck** [comment]
@g-pan just one minor comment.

File: docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml (hunk 1 above)
Comment (JamesDeFabia) [inline docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml]: I like: "To validate a package map not yet uploaded to the environment:"

**Comment by g-pan** [comment]
@afishbeck please let me know if this change is better.

File: docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml [hunk 2]
```diff
@@ -589,33 +597,301 @@ depending on where they wind up.-->
...
+              <listitem>
+                <para>Choose the <emphasis role="bold">Target </emphasis>to
+                associate the package map with.</para>
+              </listitem>
+
+              <listitem>
+                <para>Select a <emphasis role="bold">Process Filter</emphasis>
+                from the drop list. The process filter determines which
+                physical Roxie clusters will actually load the
+                packagemap.</para>
```
Comment (JamesDeFabia) [inline docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml]: package map

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Only one minor change

**Review by afishbeck — APPROVED** [review]
@g-pan looks good.

**Comment by g-pan** [comment]
Rebased for a nice clean merge

File: docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml [hunk 3]
```diff
@@ -282,7 +282,7 @@ depending on where they wind up.-->
                   </listitem>
                 </itemizedlist>
 
-                <para>The Filter also also supports wild cards. </para>
+                <para>The Filter also also supports wild cards.</para>
```
Comment (miguelvazq) [inline docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml]: You have an extra "also" in here

**Comment by miguelvazq** [comment]
FYI ECLWAPkgParts01.jpg will not have some of the fields in them after https://github.com/hpcc-systems/HPCC-Platform/pull/10094 is merged.

**Comment by miguelvazq** [comment]
Looks fine other than my image comment.

**Comment by g-pan** [comment]
Added https://track.hpccsystems.com/browse/HPCC-17827 for image update.

**Pull request merged**